test/metabase/lib/remove_replace_test.cljc

```diff
@@ -423,6 +429,9 @@
     (is (not= breakouts replaced-breakouts))
     (is (= 2 (count replaced-breakouts)))
     (is (= (second breakouts) (second replaced-breakouts)))
+    (testing "preserves the :ident"
+      (is (=? (map lib.options/ident breakouts)
+              (map lib.options/ident replaced-breakouts))))
```

Comment:
Just curious: is `=?` needed here or would `=` also work?

Reply:
Oh, `=` would work in this case. I use `=?` out of habit. Mostly because it doesn't usually matter and I'd rather not have to think about it. `=?` gives nicer diffs than `=` with more complex values. (I think for seqs of strings `=` would be as good, but it's true in general.)